easybuild/easyconfigs/c/CASTEP/CASTEP-22.11-foss-2022a.eb

```diff
@@ -0,0 +1,46 @@
+easyblock = 'ConfigureMake'
+
+name = 'CASTEP'
+version = '22.11'
+
+homepage = 'http://www.castep.org'
+description = """
+CASTEP is an electronic structure materials modelling code based on density
+functional theory (DFT), with functionality including geometry optimization
+molecular dynamics, phonons, NMR chemical shifts and much more.
+"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+# CASTEP is proprietary software, available under a free-of-charge license for academic use only.
+# Visit http://www.castep.org and navigate to "Getting Castep" to apply for a license.
```

Comment:
download_instructions = """CASTEP is proprietary software, available under a free-of-charge license for academic use only.
Visit http://www.castep.org and navigate to "Getting Castep" to apply for a license."""